tests/test_spectral.py

```diff
@@ -20,6 +21,14 @@ def test_jonswap_2():
     assert_almost_equals(FREQ[np.argmax(E)], 1/4.)
 
 
+def test_jonswap_with_arrays():
+    '''Test computation of jonswap spectrum using arrays as input'''
+    E = jonswap(FREQ, Hm0=np.array([1.0, 1.0]), Tp=np.array([4.0, 4.0]),
+                method='goda', normalize=True)
+    assert_almost_equals(np.trapz(E[:, 0], FREQ), 1/16.)
```

Comment:
This test is a bit deceiving. First we should check if we indeed get 2 dimensions and the second dimension has a length 2. So:

```
assert_equal(E.ndims, 2)
assert_equal(E.shape[1], 2)
```